### DISCUSSION TITLE
Integrate some of the ideas from https://nachhaltiges-webdesign.jetzt/

### DISCUSSION THREAD

**Discussion by neurolabs** [sandstorm/awesome-sustainable-software#2] [Ideas]
Thanks for this wonderful compilation. At https://nachhaltiges-webdesign.jetzt/, there is some more ideas that could augment the ideas you present here.

**Comment by LeonardFollner** [comment]
Hej @neurolabs,
Thank you very much for your suggestion. We just updated the list and already have contributions – how exciting. 🥳   
I incorporated some of the more technical measures from [nachhaltiges-webdesign.jetzt](https://nachhaltiges-webdesign.jetzt/) into the list.
